### PR TITLE
OH FUCK CAEL IS HERE

### LINKED ISSUE #3328: Diona node pod bug finally solved!
It seems since bees can make the trays overflow in nutrament with 11/10, this doesn't affect any plants except for diona nodes. For some reason if the nutrament levels go above 10, the pod instantly vanishes. I figured out the cause to this when every time I grew diona pods around my bees they vanished, but I grew one in the back and it finally was successful. My partner also grew them before I planted the bees and they work, so bees are what is breaking the pods.

### LINKED ISSUE #4127: (WEB REPORT BY: icycharm REMOTE: 198.245.63.50:7777) Disappearing Diona Plants
>Revision (Should be above if you're viewing this from ingame!)

>General description of the issue
Diona replica pods diappear before maturing. To clarify, they do not die, they do not result in a dead diona nymph, they just straight-up disappear.

>What you expected to happen
I expected some diona nymphs and so were the ghosts flickering my lights

>What actually happened
Wasted seeds

>Steps to reproduce if possible
Plant some diona seeds, get nothing

### LINKED ISSUE #5613: Bees kill diona pods.
bee watch enters its 4th month

https://github.com/d3athrow/vgstation13/issues/4127
https://github.com/d3athrow/vgstation13/issues/3328
https://github.com/d3athrow/vgstation13/issues/3182
https://github.com/d3athrow/vgstation13/issues/4799

fix it *please*

### PR DESCRIPTION
**Pull request by PJB3005**
Should fix #5613 

fixes #4127
fixes #3328

### CODE REVIEW

**Comment by Intigracy** [comment]
fixes #4127
fixes #3328

File: code/game/machinery/bees_apiary.dm [hunk 1]
```diff
@@ -183,7 +183,9 @@
 		for(var/obj/machinery/portable_atmospherics/hydroponics/H in view(7, src))
 			if(H.seed && !H.dead && prob(owned_bee_swarms.len * 10))
 				src.nutrilevel++
-				H.nutrilevel++
+				if(H.nutrilevel < 100)
```
Comment (RemieRichards) [inline code/game/machinery/bees_apiary.dm]: ```H.nutrilevel = min(H.nutrilevel++, 100)```

Comment (PJB3005) [inline code/game/machinery/bees_apiary.dm]: works too, though I assumed an if is faster than a min

Comment (RemieRichards) [inline code/game/machinery/bees_apiary.dm]: if byond fucked up implementing min, I'll be sad.

Comment (PJB3005) [inline code/game/machinery/bees_apiary.dm]: I actually did some tests on the matter.

Our implementation of Clamp()

``` DM
#define Clamp(x, y, z) 	(x <= y ? y : (x >= z ? z : x))
```

is faster than the following:

``` DM
#define Clamp(x, y, z) min(max(x, z), y)
```

Comment (RemieRichards) [inline code/game/machinery/bees_apiary.dm]: that's preprocessor macros though.
test ```min()``` and ```if(num < min)```

**Comment by Probe1** [comment]
who is cael

does he break things

**Comment by PJB3005** [comment]
@Probe1 he's an oldcoder who used to code for bay.

He was terrible.

**TERRIBLE**

**Pull request merged**